assets/js/modules/analytics/components/common/PropertySelect.js

```diff
@@ -35,15 +35,22 @@ import { trackEvent } from '../../../../util';
 const { useSelect, useDispatch } = Data;
 
 export default function PropertySelect() {
-	const { accountID, properties, hasResolvedProperties } = useSelect( ( select ) => {
+	const {
+		accountID,
+		properties,
+		hasStartedFetchingProperties,
+		hasResolvedProperties,
+	} = useSelect( ( select ) => {
 		const data = {
 			accountID: select( STORE_NAME ).getAccountID(),
 			properties: [],
+			hasStartedFetchingProperties: false,
 			hasResolvedProperties: false,
```

Comment:
See above, we might be able to simplify this via `isResolving`.

Reply:
Updated to use `isResolving`.